public/v4/apps/node-exporter.yml

```diff
@@ -0,0 +1,33 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: prom/node-exporter:$$cap_version
+        restart: always
+        volumes:
+            - /var/run/docker.sock:/var/run/docker.sock:ro
```

Comment:
The mounting modes (`ro`) is not being consumed by CapRover. I'll remove after merge.